src/pages/Actions/ActionEvent/EventCreate/EventCreate.tsx

```diff
@@ -208,14 +220,23 @@ export default function ({ onCreate, setShowEventCreateDrawer }: Props) {
       </Row>
       <Row justify="start" gutter={[12, 0]}>
         <Col md={24} xs={24}>
-          <Form.Item label="Дата початку" name="EventDateStart" className={classes.formItem} rules={[{ required: true, message: emptyInput() }]}>
+          <Form.Item label="Дата початку" name="EventDateStart" className={classes.formItem}
+            rules={[
+              { required: true, message: emptyInput() },
```

Comment:
please format this piese of code like this: 
```suggestion
              { 
                required: true, 
                message: emptyInput() 
              },
```